### ISSUE TITLE
Chrome Password no longer autopopulating

### ISSUE THREAD

**Issue by Foster4g** [pi-hole/web#2418]
### Versions

- Pi-hole:  5.13
- AdminLTE:  5.18.2
- FTL:  5.16

### Platform

- OS and version:  Bebian Buster

- Platform:  Raspberry Pi

Chrome 107.0.5304.88 (64 Bit)

### Expected behavior

When I go to pihole web interface, password autopopates or is selectable. worked in prior versions of pihole. 

### Actual behavior / bug

no longer populates password. Did notice the site chagned so clared out of chrome and logged in and added, but does not popuplate. 

### Steps to reproduce

load pihole web ui
password does not prompt
have to go to chrome setttings password to find password
to copy and paste, to log in. 

![image](https://user-images.githubusercontent.com/117558337/200155390-c8143f6a-9fd8-4780-a684-1126cf033943.png)

**Comment by yubiuser** [comment]
I can reproduce the issue here. I'm not sure if this is Pi-hole related or browser related, because it's working fine in firefox

**Comment by yubiuser** [comment]
I got it working by changing

https://github.com/pi-hole/AdminLTE/blob/c2afe4221ac275a1c082e1d8e14ccbb6113b0e7b/login.php#L45

to

```
<input type="text" id="username" value="pi.hole" hidden>
<input type="password" id="loginpw" name="pw" class="form-control" placeholder="Password" autocomplete="current-password" autofocus>
```

and providing a "fake" username.
But I'll ask input of @rdwebdesign if he knows a better solution.

**Comment by yubiuser** [comment]
@Foster4g 

Please try if `pihole checkout web login` solves the issue for you. You can go back on track anytime and before the next Pi-hole release with `pihole checkout web master`

**Comment by Foster4g** [comment]
Confirmed issue is resolved with new update. I did have to remove the autofill password and log in agian for it to be prompted to save, but then it popualted correctly after logging out.

Thank you!

**Comment by rdwebdesign** [comment]
This seems like a Chrome bug (the same happens with Edge).
It doesn't allow to save a password without an username.

For both browsers (Chrome and Edge) I wasn't able to save the password using the normal procedure. I typed the password, the browser showed a pop up to save the password, but nothing was saved.

BUT the password will be saved **if I add an username** (any non-empty username).

I was also able to manually create an entry (Settings > Profile > Passwords > Add password button) using any non-empty user name. I was able to use the manually created entry to login.

This worked for Chrome and Edge.
